pallets/pallet-bonded-coins/src/utils.rs

```diff
@@ -0,0 +1,15 @@
+use sp_arithmetic::ArithmeticError;
+use substrate_fixed::traits::ToFixed;
+
+use crate::{Config, CurveParameterTypeOf};
+
+pub fn convert_balance_to_parameter<T: Config>(
+	x: u128,
+	denomination: &u8,
```

Comment:
`u8` implements copy. It's faster to take it by value than to take a reference to it.